src/server.ts

```diff
@@ -10,6 +13,21 @@ function getRovers(res: any) {
       .then((e) => res.send(e.data));
   }
 
+  function getPhotos(rovername: rover, cameratype: camera, res: any) {
+    let data = axios
+      .get(
+        `https://api.nasa.gov/mars-photos/api/v1/rovers/${rovername}/photos?sol=1000&camera=${cameratype}&api_key=6R1WgjArW8R3yvTqvtv1iljThtiB9ANQVIn01PJm`
+      )
+      .then((e) => {
+        let photos = e.data.photos; // list of photo objects (list of strings technically)
+        photos = photos.map((pic: photo) => {
+          return pic.img_src; //.jpg url
+        });
+        console.log(Object.keys(photos).length)
+        photos = Object.keys(photos).length === 0 ? 'Rover does not have this camera type' : photos
+        res.send(photos); // sends list of photo urls

```

Comment:
Another small thing you could do to improve readability here (And remove comments) is to use multiple consts instead of changing the photo variable every time. e.g.

`const apiPhotos: Photo[] = data`

`const imageUrls: string[] = apiPhotos.map()`

Directly doing the ternary in the res.send:
`res.send(imageUrls.length === 0 ? 'blah' : imageUrls )`

By defining the types too you can make it clear what each list is along the chain, I think that after you do your map you just get a list of strings rather than a dictionary so you don't need to use Object.keys() :D